packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md

```diff
@@ -357,7 +372,11 @@ The `active` attribute in a user profile represents the user's current status.
 
 Other updates to attributes in a user profile should be handled through a PUT operation.
 
-To deactivate users, the functionality needs to be enabled in Okta. In the Okta Admin Console, under the **Provisioning** tab, click **To App** and **Edit**. In the **Deactivate Users** option, click **Enable** and then **Save**.
+To deactivate users, the functionality needs to be enabled in Okta. In the Okta Admin Console:
+
+* Select your SCIM application from your list of applications
```

Comment:
So, these are steps that need to be numbered and need periods at the end of each.